CONTRIBUTING.rst

```diff
@@ -139,8 +133,25 @@ See in the sections above for the right way to do this.
 Setting up the environment
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Maybe this part should be at the top?

Reply:
Do you mean this section should be before "How can I contribute?

Reply:
Yes, right before seems better I think

Reply:
But don't you think it will be confusing for users only interested in submitting an issue, for example?

Reply:
Ok, actually let's keep it this way..